packages/ui/src/ui/containers/OpenQueryButton/OpenQueryButton.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import cn from 'bem-cn-lite';
+import {Button} from '@gravity-ui/uikit';
+import {useDispatch} from 'react-redux';
+
+import Icon from '../../components/Icon/Icon';
+import {useQueryWidgetSidePanel} from '../../pages/query-tracker/QueryWidget/side-panel';
+import {QueryEngine} from '../../pages/query-tracker/module/engines';
+import {createQueryFromTablePath} from '../../pages/query-tracker/module/query/actions';
+import {createNewQueryUrl} from '../../pages/query-tracker/utils/navigation';
+
+import './OpenQueryButton.scss';
+
+const b = cn('yt-open-query-button');
+
+export type OpenQueryButtonProps = {
```

Comment:
Are you sure you need an export here? I don't see any other places to use it.

Reply:
It is a good pattern to export a component with its props